docs/src/_tutorials/logging.md

```diff
@@ -0,0 +1,284 @@
+
+loggers:
+  somespecific.module.logger: # if you want debug logs for a specific named logger or module
+    level: DEBUG
+    handlers: [console]
+    propogate: no
+  urllib3: # for example hide all urllib3 debug logs
+    level: WARNING
+    handlers: [console, meltano_log]
+    propogate: no
+```
+
+## Local development config example
+
+While working with meltano locally it's sometimes nice to have more terse logging on the console, but still have DEBUG
+level info written to a log file behind scenes incase you need to debug something. To accomplish that, you can use a
+file like:
+
+```yaml
+version: 1
+disable_existing_loggers: false
+
+formatters:
+  structured_colored:
+    (): meltano.core.logging.console_log_formatter
+    colors: True
+  json:
+    (): meltano.core.logging.json_formatter
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: INFO
+    formatter: structured_colored
+    stream: "ext://sys.stderr"
+  file:
+    class: logging.FileHandler
+    level: DEBUG
+    filename: meltano.log
+    formatter: json
+
+root:
+  level: DEBUG
+  propagate: yes
+  handlers: [console, file]
+```
+
+To have it be even more terse, you can use level `WARN` instead of `INFO`. In the case of something like a successful
+`meltano run` invocation this would produce no output at all.
+
+## A generic starting config for log management providers
+
+Most logging management tools will readily accept structured logs delivered in JSON format. As such, when all else fails
+configuring Meltano to log in JSON format is a good first step.
+
+For example, to log to a file called `meltano.log` in JSON format, while also reporting WARNING lines and above on
+the console you could use the following `logging.yaml` config:
+
+```yaml
+version: 1
+disable_existing_loggers: false
+
+formatters:
+  structured_plain:
+    (): meltano.core.logging.console_log_formatter
+    colors: False
+  json:
+    (): meltano.core.logging.json_formatter
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: WARNING
+    formatter: structured_plain
+    stream: "ext://sys.stderr"
+  file:
+    class: logging.FileHandler
+    level: INFO
+    filename: meltano.log
+    formatter: json
+
+root:
+  level: DEBUG
+  propagate: yes
+  handlers: [console, file]
+```
+
+If instead you wanted the console output to log in JSON format because your logging solution is capturing output directly
+you could use the following config:
+
+```yaml
+version: 1
+disable_existing_loggers: false
+
+formatters:
+  json:
+    (): meltano.core.logging.json_formatter
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    formatter: json
+    stream: "ext://sys.stderr"
+
+root:
+  level: INFO
+  propagate: yes
+  handlers: [console]
+```
+
+## Datadog logging config
+
+For Datadog logging you have a couple of options. The easiest approach maybe to simply log to a file in json format and
```

Comment:
```suggestion
You have a couple options for structuring logs for Datadog. The easiest approach may be to log to a file in JSON format and
```

Reply:
This is less "structuring logs for Datadog" and more "configuring logs for Datadog". Mind if I roll with that phrasing ?